bin/razor

```diff
@@ -15,25 +15,12 @@ def die(message = nil)
 end
 
 begin
```

Comment:
I don't think we need this begin block anymore.

Reply:
I think you're right. :D